apps/src/sites/studio/pages/essential.js

```diff
@@ -1,9 +1,16 @@
 import {isUnsupportedBrowser} from '@cdo/apps/util/browser-detector';
 
-// Prevent filtered errors from being passed to New Relic.
+// Prevent filtered errors from being passed to New Relic. Return true when New
+// Relic should ignore the error.
 if (window.newrelic) {
   window.newrelic.setErrorHandler(function (err) {
     // Remove errors from unsupportenewrelicnd IE versions
-    return !!isUnsupportedBrowser();
+    if (isUnsupportedBrowser()) {
+      return true;
+    }
+    // Filter out errors with no stack trace (these aren't actionable yet)
+    if (!err.stack || err.stack.length < 1) {
+      return true;
+    }
```

Comment:
Why are we implicitly returning `undefined` instead of explicitly returning `false`?  While this matches the documented behavior of the handler (if returned true X, otherwise Y) I think it's less surprising if this function always returns true or false (and might avoid future breakage if there's a subtle change in the API).